app/src/main/java/ip/team13/petowner/ui/login/LoginViewModel.kt

```diff
@@ -16,6 +19,8 @@ class LoginViewModel(private val authRepository: AuthRepository) : ViewModel() {
 
         if (!userEmail.isValidEmail()) return
 
-        authRepository.login(userEmail, userPassword)
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
Can we use the `viewModelScope` `lifecycle` extension from `androidx` instead of `CoroutineScope()`? It ensures the calls are cancelled whenever the VM gets destroyed. Same applies for register screen.